sites/site-utilities/src/web-components/pragma.ts

```diff
@@ -5,5 +5,4 @@ import val from "@skatejs/val";
  * Define the wrapped createElement function - this should be imported and
  * declared as the JSX pragma wherever Web Components are rendered by React
  */
-
 export default val(createElement);
```

Comment:
@Ibrahimmaga we likely don't wan to check in this change here - I assume it was a test